executor/executor_xapi.go

```diff
@@ -332,7 +361,11 @@ func (e *XSelectIndexExec) doIndexRequest() (*xapi.SelectResult, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	return xapi.Select(txn.GetClient(), selIdxReq, 1)
+	concurrency := 1
```

Comment:
Index in one region may be enough for the query, set concurrency to 1 to save the work that tikv need to do.